py-polars/polars/expr/datetime.py

```diff
@@ -57,6 +57,10 @@ def truncate(
             Every interval start and period length
         offset
             Offset the window
+
+            .. deprecated:: 0.20.19
+                This argument is deprecated and will be removed in the next breaking
+                release. Instead, chain `dt.truncate()` with `.dt.offset()`.
```

Comment:
If I may jump in here, you say `.dt.offset()` here but `.dt.offset_by()` in the message below

Reply:
thanks @etiennebacher for spotting this! @TrevorWinstral fancy pulling and fixing this up?